copy/copy.go

```diff
@@ -386,6 +393,11 @@ func (c *copier) copy(ctx context.Context, src, srcComponents, target string, ov
 		if err := copyXAttrs(target, src, c.xattrErrorHandler); err != nil {
 			return errors.Wrap(err, "failed to copy xattrs")
 		}
+	} else if restoreFileTimestamp && targetFi != nil {
+		t := targetFi.ModTime()
+		if err := Utimes(target, &t); err != nil {
```

Comment:
This doesn't set the atime and mtime separately like the `copyFileInfo`. Not a problem?

Reply:
Yup, probably a problem :laughing:

Reply:
Fixed. Have extracted the timestamp portions of `copyFileInfo` into `copyFileTimestamp`, so we can call that here.